src/Microsoft.OData.Core/JsonLight/ODataJsonLightReader.cs

```diff
@@ -695,27 +788,6 @@ protected override bool ReadAtDeletedResourceStartImplementation()
             return this.ReadAtDeletedResourceStartImplementationSynchronously();
         }
 
-        /// <summary>

```

Comment:
No need for this since the implementation in the base class mirror this, i.e.,
```csharp
protected virtual Task<bool> ReadAtDeletedResourceStartImplementationAsync()
{
    return TaskUtils.GetTaskForSynchronousOperation<bool>(this.ReadAtDeletedResourceStartImplementation);
}
```